charts/hatchet-stack/values.yaml

```diff
@@ -3,6 +3,8 @@ api:
   replicaCount: 2
   image:
     repository: "ghcr.io/hatchet-dev/hatchet/hatchet-api"
+    tag: "v0.13.1"
```

Comment:
It wouldn't set the tag until I added it here. Probably `.image` overwrites everything so we need to set it again here I guess